<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>DelPreset</class>
  <widget class="QDialog" name="DelPreset">
    <property name="styleSheet">
      <string notr="true">
    background-color: rgb(42, 45, 46);
    color: white; 
      </string>
    </property>
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>369</width>
        <height>347</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Настройки принтера</string>
    </property>
    <property name="windowIcon">
      <iconset>
        <normaloff>:/img/logo-1.png</normaloff>
      </iconset>
    </property>
    <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
        <rect>
          <x>20</x>
          <y>20</y>
          <width>321</width>
          <height>271</height>
        </rect>
      </property>
      <property name="styleSheet">
        <string notr="true">background-color: rgb(36, 36, 36);</string>
      </property>
    </widget>
    <widget class="QWidget" name="">
      <property name="geometry">
        <rect>
          <x>23</x>
          <y>310</y>
          <width>321</width>
          <height>27</height>
        </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
          <widget class="QPushButton" name="edit_button">
            <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 80, 0);</string>
            </property>
            <property name="text">
              <string>Изменить</string>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="delete_button">
            <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 0, 0);</string>
            </property>
            <property name="text">
              <string>Удалить</string>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="reject_button">
            <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 80, 0);</string>
            </property>
            <property name="text">
              <string>Отмена</string>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>
  <resources/>
  <connections/>
</ui>
